src/main/java/monique/parser/Parser.java

```diff
@@ -20,136 +20,135 @@
 import monique.task.ToDo;
 
 public class Parser {
-
     private static final Set<String> commands = Set.of("list", "mark", "unmark", "bye", "/commands", "delete");
     private static final Set<String> taskTypes = Set.of("todo", "deadline", "event");
 
-    public static Command parse(String fullCommand){
+    public static Command parse(String fullCommand) {
         String firstWord = fullCommand.split(" ")[0];
         boolean hasSecondWord = fullCommand.split(" ").length >1;
         Command command = null;
 
         if (commands.contains(firstWord)) {
-            switch(firstWord) {
-                case "bye": {
-                    command = new ByeCommand();
+            switch (firstWord) {
+            case "bye": {
+                command = new ByeCommand();
+                break;
+            }
+            case "list": {
+                command = new ListCommand();
```

Comment:
Switch case format follows CS2103T format. Good job.
In the mark case, you've included the comment // minus one bc 0-based indexing, which is good for readability. However, to make the code even clearer, you might consider defining a constant, such as INDEX_OFFSET = 1, to avoid the use of hardcoded values.

Reply:
Thanks, I've fixed it!